nirc_ehr/src/org/labkey/nirc_ehr/dataentry/section/NIRCBirthFormSection.java

```diff
@@ -41,16 +44,29 @@ protected List<FieldKey> getFieldKeys(TableInfo ti)
     @Override
     public List<String> getTbarButtons()
     {
-        List<String> defaults = super.getTbarButtons();
-        defaults.remove("COPYFROMSECTION");
-        return defaults;
+        List<String> defaultButtons = super.getTbarButtons();
+
+        int idx = defaultButtons.indexOf("ADDANIMALS");
+        if (idx > -1)
+        {
+            defaultButtons.remove(idx);
+            defaultButtons.add(idx, "NIRC_ADDANIMALS");
+        }
+
+        idx = defaultButtons.indexOf("COPYFROMSECTION");
+        if (idx > -1)
+        {
+            defaultButtons.remove(idx);
+        }
```

Comment:
You should be able to just do `defaults.remove("COPYFROMSECTION");`. It will leave the list unchanged and not throw an error if it doesn't exist.

Reply:
done